.github/workflows/ios-build-test.yml

```diff
@@ -15,6 +15,16 @@ jobs:
       - name: checkout repository
         uses: actions/checkout@v3
 
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18
```

Comment:
Consider locking the node version in a `.nvmrc` file at the top of the repo and using `node-version-file`. That way other tools can setup the correct node version automatically.

Reply:
Sounds good, I'll do that, thanks!